curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-basic-javascript-by-building-a-pyramid-generator/660f46460f9c36330ebc07d8.md

```diff
@@ -0,0 +1,57 @@
+---
+id: 660f46460f9c36330ebc07d8
+title: Step 59
+challengeType: 0
+dashedName: step-59
+---
+
+# --description--
+
+The <dfn>strict equality</dfn> operator `===` is used to check if two values are equal and share the same type. As a general rule, this is the equality operator you should always use. To compare two values, you'd use a statement like `value === 8`.
```

Comment:
I don't know if we should mention first mention that JavaScript has an equality operator.

maybe we could have separate lessons for this and talk about type coercion.
And then that could lead into  strict equality and why it is preferred. 

that is what we did in the legacy curriculum here
https://www.freecodecamp.org/learn/javascript-algorithms-and-data-structures/basic-javascript/comparison-with-the-equality-operator

https://www.freecodecamp.org/learn/javascript-algorithms-and-data-structures/basic-javascript/comparison-with-the-strict-equality-operator